iothub/device/src/DirectMethod/DirectMethodResponse.cs

```diff
@@ -26,8 +27,11 @@ public DirectMethodResponse()
         /// <summary>
         /// The optional direct method payload.
         /// </summary>
+        /// <remarks>
+        /// The payload can be null or Primitive type (e.g., String, Int)/Array/List/Map/custom type.
```

Comment:
```suggestion
        /// The payload can be null or primitive type (e.g., string, int/array/list/dictionary/custom type).
```

Reply:
If we're providing examples of primitive types, why mention custom types at all? Also, should we consider adding a link to the [built-in C# types](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types)?

Reply:
I think adding a link a to built-in c# types would be an overkill.

We do need to mention custom types as its an important feature to support it